ratbag_emu/server.py

```diff
@@ -8,6 +13,19 @@
 from ratbag_emu.protocol.devices import DeviceList
 
 
+logger = logging.getLogger('ratbagemu.server')
+
+
+def _error(message, code):
+    logger.info(f'error {code}: {message}')
+    return json.dumps(message), code
+
+
+def _success(message, code):
+    logger.info(f'success {code}: {message}')
+    return json.dumps(message), code
```

Comment:
I think this should be debug. Connexion already logs the return code, I don't think the error message is relevant enough to justify having one extra log line. The error message is usually already logged by the client so I can't help but feel like this is a bit of spam. What do you think?